otx/mpa/deploy/apis.py

```diff
@@ -195,6 +195,11 @@ def helper(*args, **kwargs):
                 "input_data",
                 {"shape": (128, 128, 3), "file_path": None},
             )
+
+            # patch test_pipeline in case of missing LoadImageFromOTXDataset
+            if cfg.data.test.pipeline[0].type != "LoadImageFromOTXDataset":
+                cfg.data.test.pipeline.insert(0, mmcv.ConfigDict(dict(type="LoadImageFromOTXDataset")))
```

Comment:
Hi @eugene123tw,
What is the background for this code being added? Due to this, the data pipeline is built and throws an error even during the current `otx export`. (current export, this code works and LoadImageFromOTXDataset is called and an error is raised there.)
Is this what we need for tiling?